polaris-react/.storybook/GridOverlay/GridOverlay.css

```diff
@@ -7,18 +7,32 @@
   bottom: 0;
   right: 0;
   display: grid;
-  gap: var(--p-space-3);
-  grid-template-columns: repeat(4, 1fr);
-  padding: 0 var(--p-space-3);
+  gap: var(--p-space-4);
+  grid-template-columns: repeat(6, 1fr);
+  padding: 0 var(--p-space-4);
   margin: 0 auto;
   background-color: var(--p-action-critical);
   opacity: 0.25;
   pointer-events: none;
 }
-/* #{$p-breakpoints-sm-up} */
+
+/* $p-breakpoints-sm-up */
```

Comment:
**note (non-blocking):** I included the interpolation syntax `#{...}` to ensure these instances were matched in global searches for Polaris breakpoint alias usage e.g. `#{$p-breakpoints-`